packages/core/contracts/insured-bridge/implementation/L1_BridgePool.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+contract BridgePool {}
```

Comment:
I created this stub as it's likely the L1 contracts will have two parts:
1) a router that all users interact with
2) a pool that contains the funds for a particular bridging action.